src/app/chat/component/chat/chat.component.ts

```diff
@@ -5,6 +5,7 @@ import { Message } from '../../model/Message.model';
 import { FormControl } from '@angular/forms';
 import { SocketService } from '../../service/socket/socket.service';
 import { UserService } from '@global-service/user/user.service';
+import { EmojiService } from '@global-service/emoji/emoji.service';
 
 @Component({
```

Comment:
`ChatComponent` is not declared in any Angular module
Unresolved variable emptyChatBack in template: 26:
`<img [src]="chatIcons.emptyChatBack" alt="" />`